packages/camera-web/src/Camera/hooks/useUserMedia.ts

```diff
@@ -36,6 +36,15 @@ class InvalidStreamError extends Error {
  * @see useUserMedia
  */
 export enum UserMediaErrorType {
+  /**
+   * The camera stream couldn't be fetched because the web page does not have the permissions to access the camera.
+   * if both webpage and browser permissions are denied, this error will be returned.
+   */
+  NOT_ALLOWED_WEBPAGE = 'not_allowed_webpage',
+  /**
+   * The camera stream couldn't be fetched because the browser does not support the `getUserMedia` function.
```

Comment:
Replace error message with ``The camera stream couldn't be fetched the browser is unable to properly call the `getUserMedia` function, either because it does not support it or because the camera permissions are not granted to the browser in the device settings.``

Reply:
if permission query is not supported then we will show Not_allowed.

Reply:
Change description of error to : `The camera stream couldn't be fetched because the camera permissions are not granted to the browser in the device settings.`